src/Microsoft.PowerApps.TestEngine.Tests/PowerFx/Functions/SelectFunctionTests.cs

```diff
@@ -75,7 +76,9 @@ public void SelectOneParamFunctionTest()
             var result = selectFunction.Execute(recordValue);
             Assert.IsType<BlankValue>(result);
             MockPowerAppFunctions.Verify(x => x.SelectControlAsync(It.Is<ItemPath>((item) => item.ControlName == recordValue.Name)), Times.Once());
+            LoggingTestHelper.VerifyLogging(MockLogger, "------------------------------\n\n" + "Executing Select function.", LogLevel.Information, Times.Once());
             Assert.Equal(1, updaterFunctionCallCount);
+            LoggingTestHelper.VerifyLogging(MockLogger, "Assert failed. Property is not equal to the specified value.", LogLevel.Information, Times.Once());
```

Comment:
This assert is wrong looks like, ` Successfully finished executing Select function.`  would be expected message here